packages/EasyCore/tests/Doctrine/Dispatchers/DeferredEntityEventDispatcherTest.php

```diff
@@ -0,0 +1,236 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyCore\Tests\Doctrine\Dispatchers;
+
+use EonX\EasyCore\Doctrine\Dispatchers\DeferredEntityEventDispatcher;
+use EonX\EasyCore\Doctrine\Events\EntityCreatedEvent;
+use EonX\EasyCore\Doctrine\Events\EntityUpdatedEvent;
+use EonX\EasyCore\Interfaces\DatabaseEntityInterface;
+use EonX\EasyCore\Tests\AbstractTestCase;
+use Prophecy\Argument;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+
+/**
+ * @covers \EonX\EasyCore\Doctrine\Dispatchers\DeferredEntityEventDispatcher
+ */
+final class DeferredEntityEventDispatcherTest extends AbstractTestCase
```

Comment:
Pls cover the disable and enable methods.

Reply:
Done